README.md

```diff
@@ -1,13 +1,47 @@
 # falcor-postman
 
-falcor-postman
+A tool that enables you to exercise your Express.js app Falcor's endpoint and therefore helps you to validate your queries.
+
+## Usage
+
+Npm install using the following command:
 
 ```
-npm i
+npm i -S falcor-postman
```

Comment:
For the sake of documentation, I will go with the full form: 
`npm install --save falcor-postman`